examples/utils/webpack-fluid-loader/src/odspUrlResolver.ts

```diff
@@ -68,16 +68,16 @@ export class OdspUrlResolver implements IUrlResolver {
 
 	public async createCreateNewRequest(fileName: string): Promise<IRequest> {
 		const filePath = "/r11s/";
-		const driveItem = await getDriveItemByRootFileName(
+		const { driveId } = await getDriveItemByRootFileName(
 			this.server,
 			"",
-			filePath,
+			"/",
```

Comment:
This is just a bug fix--previous version of this file made examples fail for people who didn't have an `/r11s/` folder in their drive already (which obviously wasn't true for the test accounts either).

Reply:
bit of a head-scratcher to find :P